ucsmsdk/utils/ucstechsupport.py

```diff
@@ -221,7 +468,7 @@ def get_ucs_tech_support(handle,
             sys_debug_tech_support_cmd_opt.major_opt_type = \
                 SysdebugTechSupportCmdOptConsts.MAJOR_OPT_TYPE_SERVER
```

Comment:
@waghswapnil good point. I noticed that the original code was doing it so. But I will try changing it now.